diff -ruN --exclude=node_modules --exclude=deployments --exclude=governance-scripts ../contracts/governance/Managed.sol governance/Managed.sol
--- ../contracts/governance/Managed.sol	2023-02-09 12:38:29.000000000 -0500
+++ governance/Managed.sol	2023-02-17 20:32:17.000000000 -0500
@@ -28,17 +28,17 @@
     /// Controller that contract is registered with
     IController public controller;
     /// @dev Cache for the addresses of the contracts retrieved from the controller
-    mapping(bytes32 => address) private _addressCache;
+    mapping(bytes32 => address) public _addressCache;
     /// @dev Gap for future storage variables
     uint256[10] private __gap;
 
     // Immutables
-    bytes32 private immutable CURATION = keccak256("Curation");
-    bytes32 private immutable EPOCH_MANAGER = keccak256("EpochManager");
-    bytes32 private immutable REWARDS_MANAGER = keccak256("RewardsManager");
-    bytes32 private immutable STAKING = keccak256("Staking");
-    bytes32 private immutable GRAPH_TOKEN = keccak256("GraphToken");
-    bytes32 private immutable GRAPH_TOKEN_GATEWAY = keccak256("GraphTokenGateway");
+    bytes32 public immutable CURATION = keccak256("Curation");
+    bytes32 public immutable EPOCH_MANAGER = keccak256("EpochManager");
+    bytes32 public immutable REWARDS_MANAGER = keccak256("RewardsManager");
+    bytes32 public immutable STAKING = keccak256("Staking");
+    bytes32 public immutable GRAPH_TOKEN = keccak256("GraphToken");
+    bytes32 public immutable GRAPH_TOKEN_GATEWAY = keccak256("GraphTokenGateway");
 
     // -- Events --
 
diff -ruN --exclude=node_modules --exclude=deployments --exclude=governance-scripts ../contracts/rewards/RewardsManager.sol rewards/RewardsManager.sol
--- ../contracts/rewards/RewardsManager.sol	2023-02-09 12:40:00.000000000 -0500
+++ rewards/RewardsManager.sol	2023-02-17 20:24:01.000000000 -0500
@@ -31,7 +31,7 @@
 contract RewardsManager is RewardsManagerV4Storage, GraphUpgradeable, IRewardsManager {
     using SafeMath for uint256;
 
-    uint256 private constant FIXED_POINT_SCALING_FACTOR = 1e18;
+    uint256 public constant FIXED_POINT_SCALING_FACTOR = 1e18;
 
     // -- Events --
 
